app/src/main/java/com/xinto/opencord/domain/attachment/DomainAttachment.kt

```diff
@@ -0,0 +1,75 @@
+package com.xinto.opencord.domain.attachment
+
+import com.xinto.opencord.db.entity.message.EntityAttachment
+import com.xinto.opencord.rest.dto.ApiAttachment
+
+interface DomainAttachment {
```

Comment:
we need to seal these interfaces and move implementations *inside* them

Reply:
otherwise one could accidentally extend these interfaces

Reply:
> move implementations inside them

then you missed the entire point of this pr :trollface: 
the point of moving them to different files is so we don't get huge ass files with no organization
especially as domain message model grows

Reply:
either way, its not a big deal
imo sealed isnt really that move helpful

Reply:
> > move implementations inside them
> 
> then you missed the entire point of this pr :trollface: the point of moving them to different files is so we don't get huge ass files with no organization especially as domain message model grows

guhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh

Reply:
sure fine then it's not a huge problem really